composer.json

```
@@ -18,7 +18,7 @@
     ],
     "require": {
         "php": "~8.1.0 || ~8.2.0",
-        "psr/http-message": "^1.0",
+        "psr/http-message": "^1.1 || ^2.0",
```

```suggestion
        "psr/http-message": "^1.0 || ^2.0",
```
I know that the issue says `^1.1`, but it should be `^1.0`.